packages/pyright-internal/src/languageService/completionProvider.ts

```diff
@@ -1659,6 +1660,13 @@ export class CompletionProvider {
                             } else if (primaryDecl.type === DeclarationType.Function) {
                                 // @property functions
                                 documentation = getFunctionDocStringFromDeclaration(primaryDecl, this._sourceMapper);
```

Comment:
I could also just force this to be the first declaration

`getFunctionDocStringFromDeclaration(declarations[0], this._sourceMapper);`